### DISCUSSION TITLE
znap pull prints errors -- why?

### DISCUSSION THREAD

**Discussion by nimish** [marlonrichert/zsh-snap#111] [General]
`znap pull` results in 

```
.znap.pull.task:13: bad math expression: operand expected at end of string
..znap.pull.task:13: bad math expression: operand expected at end of string
```

the repos seem to be pulled, however!

**Comment by nimish** [comment]
Blew away my `.zsh` folder and reinstalled znap, went away. Weird, must have been some caching issue

**Reply by marlonrichert** [reply]
Well, if it happens again, don't be afraid be file a bug report. 🙂

**Reply by marlonrichert** [reply]
Well, how about that? I was able to reproduce it just now. It happens when a repo is not currently on a branch, for example, when in the middle of a rebase. That's fixed now: 252dc6a

--- 

PS: If you enjoy using my software, please consider [sponsoring me](https://github.com/sponsors/marlonrichert). 🙂

**Reply by nimish** [reply]
Solid. Tossed a few $ your way, have been using a bunch of your libs for zsh. Great work!
